### PR TITLE
Various small documentation related changes

### PR DESCRIPTION
**Pull request by hhorak**
Use maintainer label consistently
Few edits in top-level README
Using usage label and command

### CODE REVIEW

**Comment by hhorak** [comment]
Also removes release from CentOS and RHEL Dockerfiles.

**Comment by pkubatrh** [comment]
lgtm, thanks! Merging

File: README.md [hunk 1]
```diff
@@ -32,29 +33,35 @@ Choose either the CentOS7 or RHEL7 based image:
...
+    ```
+
     To build a RHEL7 based image, you need to run Docker build on a properly
     subscribed RHEL machine.
 
     ```
     $ git clone --recursive https://github.com/sclorg/postgresql-container.git
     $ cd postgresql
-    $ make build TARGET=rhel7 VERSIONS=9.5
+    $ make build TARGET=rhel7 VERSIONiS=9.6
```
Comment (praiskup) [inline README.md]: Typo 'VERSIONiS'.

Comment (hhorak) [reply README.md]: Fixed, thanks for spotting this.

**Pull request merged**